.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I'm a big fan of prettier, and also very enthusiastic about the core of this PR.  But if you don't mind splitting into 2 PRs, it would make the Github Security Advisories part easier to read, without the prettier changes, IMO.